src/feed_handler.py

```diff
@@ -261,3 +265,25 @@ def get_books(self) -> Dict[int, lob.LimitOrderBook]:
         Returns the books stored in this feed handler.
         """
         return self.books
+
+    def compare(self, other: StandardFeedHandler) -> bool:
+        """
+        Compares the two standard feed handlers. Returns a
+        boolean True if the feed handlers are equivalent, otherwise
+        error lgos out inconsistencies.
+        """
+        failed = False
+        self_books = self.get_books()
+        other_books = other.get_books()
+        if len(self_books) != len(other_books):
+            logging.error(f"Mismatched book clob pair ids: \n{self_books.keys()}\n{other_books.keys()}")
+            return False
+
+        for cpid in self_books.keys():
+            self_book = self_books[cpid]
+            other_book = other_books[cpid]
+            comparison_failed = self_book.compare_books(other_book)
+            if comparison_failed:
+                failed = True
+        # TODO compare subaccounts
```

Comment:
remove todo?

Reply:
Think it has to be done later, will keep after you add this functionality